docs/runtime_CLI.md

```diff
@@ -469,11 +469,25 @@ TODO: reset_state
 TODO: serialize_state
 TODO: set_crc16_parameters
 TODO: set_crc32_parameters
-TODO: set_queue_depth [simple_switch_CLI only]
-TODO: set_queue_rate [simple_switch_CLI only]
 TODO: shell
 ```
 
+### set_queue_depth [simple_switch_CLI only]
+```
+RuntimeCmd: help set_queue_depth
+Set depth of one / all egress queue(s): set_queue_depth <nb_pkts> [<egress_port> [<priority>]]
+```
+
+This command will set queue's maximum depth for a priority queue that is identified by egress_port and priority. If priority is not given, this command will set queue's maximum depth for all queues in the given egress_port. If both priority and egress_port are not given, this command will set queue's maximum depth for all queues. The priority mentioned here is the same as qid in the queueing_metadata.
```

Comment:
Please be a little bit more explicit about the units of <nb_pkts> in the text, e.g. something like "will set the queue's maximum depth, in units of number of packets, for a priority queue ..."

Reply:
I add more description in the following commit, thanks for the input!